example/src/components/RoomParticipant.tsx

```diff
@@ -0,0 +1,94 @@
+import { BrandColors, AdditionalColors, TextColors } from '@colors';
+import * as Membrane from '@jellyfish-dev/react-native-membrane-webrtc';
+import { getShortUsername } from '@utils';
+import React from 'react';
+import { View, StyleSheet, ViewStyle } from 'react-native';
+
+import { Icon } from './Icon';
+import { NoCameraView } from './NoCameraView';
+import { Typo } from './Typo';
+
+type RoomParticipantProps = {
+  participant: Membrane.Participant;
+  pStyle: ViewStyle[];
```

Comment:
Why `pStyle` and not just `style`? 
Also for styles like positioning etc. you can just wrap `RoomParticipant` in a styled view and avoid passing a style prop, wouldn't it be easier?
In general the component shouldn't care about its position and margins and the parent component cares about position and margins of the child component. That makes components more reusable.

Reply:
Thanks for this comment, I agree it makes sens to keep the margin styling out of the component, should be ok now.